### ISSUE TITLE
Broken in Firefox

### ISSUE THREAD

**Issue by johngeorgewright** [jakiestfu/Snap.js#6]
Just tried implementing Snap on a site and I'm getting this error:

```
e.srcElement is undefined
[Break On This Error] if (e.srcElement.dataset.snapIgnore === "true") {
main.js (line 4957)
```

The same problem on your demo site :(

**Comment by jrussell-ivantage** [comment]
Just saw this as well. In FF `e.target` should be used.

**Comment by jakiestfu** [comment]
Thanks for the heads up. I will also be doing CSS3 prefixes as well

**Comment by jakiestfu** [comment]
Fixed: 57e94a2ebce58a9b90d4f93274f8cc1034149921
